lab_1/main.py

```diff
@@ -13,7 +13,16 @@ def tokenize(text: str) -> list:
     e.g. text = 'The weather is sunny, the man is happy.'
     --> ['the', 'weather', 'is', 'sunny', 'the', 'man', 'is', 'happy']
     """
-    pass
+    if type (text) == str:
+        clean_text = ''
+        for sign in text:
+            if sign.lower() in 'abcdefghijklmnopqrstuvwxyz ':
+                clean_text += sign.lower()
+        tokens = clean_text.split()
+        return (tokens)
+    else:
```

Comment:
you can remove "else", if "return tokens" works, the function will stop its work anyway